CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog TYPO3 Crawler widget
+
+## Version 1.0.0
```

Comment:
I would start with a 0.0.1 version 1.0.0 indicates that it's stable already. I would wait a little with "claiming" that.

Reply:
I changed the version to 0.0.1